### ISSUE TITLE
Loop Midi-File

### ISSUE THREAD

**Issue by FunkeMT** [FunkeMT/ScalaKata2#5]
After start playing the Midi-File -> play it in a __loop__

**Comment by FunkeMT** [comment]
Workaround

```js
    // Loop Music
    Player.on('endOfFile', function() {
        console.log("endOfFile");

        // temporary workaround to loop file (call play() from another context)
        setTimeout(repeat, DOM_MIN_TIMEOUT_VALUE);
    });
```

**Comment by FunkeMT** [comment]
fixed by 74b48063
